src/app/util/mock/attribute-storage.cpp

```diff
@@ -351,9 +350,15 @@ CHIP_ERROR ReadSingleMockClusterData(FabricIndex aAccessingFabricIndex, const Co
     // Attribute 4 acts as a large attribute to trigger chunking.
     if (aPath.mAttributeId == MockAttributeId(4))
     {
-        AttributeValueEncoder::AttributeEncodeState state =
-            (apEncoderState == nullptr ? AttributeValueEncoder::AttributeEncodeState() : *apEncoderState);
-        AttributeValueEncoder valueEncoder(aAttributeReports, aAccessingFabricIndex, aPath, dataVersion, false, state);
+        AttributeEncodeState state;
+        if (apEncoderState != nullptr)
+        {
+            state = *apEncoderState;
+        }
+        Access::SubjectDescriptor subject;
+        subject.fabricIndex = aAccessingFabricIndex;
+
+        AttributeValueEncoder valueEncoder(aAttributeReports, subject, aPath, dataVersion, false /* isFabricFiltered */, state);
```

Comment:
```suggestion
        AttributeValueEncoder valueEncoder(aAttributeReports, subject, aPath, dataVersion, /* isFabricFiltered = */ false , state);
```

Reply:
I did not do this because AttributeValueEncoder uses the `aCamelCase` convention for arguments and clang-tidy enforces that `/* ...  = */` arguments match ... and `aIsFabricFiltered` felt uglyer than `isFabricFiltered` .

Updated to the `=` style with appropriate name, however overall I am not a fan of `a...` naming style that we inherited.